client/src/global/components/FatalError/index.js

```diff
@@ -87,84 +75,108 @@ class FatalError extends PureComponent {
       <Redirect
         to={{
           pathname: url,
-          state: { projectAuthorizationError: this.error }
+          state: {
+            showLogin: true,
+            postLoginRedirect: redirectPath,
+            setIsRedirecting: true
```

Comment:
Need to remove `setIsRedirecting` here. I tried using state here to update state in the Manifold Container to prevent the double redirect, and I'm still not sure why it didn't work. Only the error check worked.